apps/site/assets/css/_event-details.scss

```diff
@@ -2,6 +2,9 @@
 
   .c-cms__title {
     margin-top: 2.5rem;
+    @media (min-width: 800px) {
```

Comment:
[Comment/suggestion for the future] We have media breakpoints defined in `_variables.scss` that you could use here. Looks like
```css
@include media-breakpoint-up(md) { ... }
```
might be what you need here?